test/e2e/e2e-test/docker/kafka/Dockerfile.provider

```diff
@@ -27,13 +27,15 @@ RUN mv apache-skywalking-apm-bin/* . || true
 
 RUN sed -i 's/# plugin.kafka.bootstrap_servers/plugin.kafka.bootstrap_servers/g' agent/config/agent.config
 RUN  echo 'plugin.kafka.get_topic_timeout=${SW_GET_TOPIC_TIMEOUT:60}' >> agent/config/agent.config
+RUN  echo 'plugin.kafka.producer_config[compression.type]=snappy' >> agent/config/agent.config
```

Comment:
Don't we need to activate compress on the server side? Is it auto detected?

Reply:
We did not do any configuration on the kafka-server compression of my test case

Reply:
Yes, but the case is still passed. This line means open compression at the agent side, right?

Reply:
> Don't we need to activate compress on the server side? Is it auto detected?

Yes, auto detected by default in kafka-broker.

![image](https://user-images.githubusercontent.com/14371345/129482985-176d4f18-4e01-49d2-8616-d4c9f53c148c.png)